site/docs-md/apis/filesystem/index.md

```diff
@@ -15,6 +15,8 @@ leave out the `directory` param to use a full file path.
 
 ## Example
 
+See also [Using the Capacitor Filesystem API to Store Photos](https://www.joshmorony.com/using-the-capacitor-filesystem-api-to-store-photos/).
```

Comment:
this one doesn't seem up to date neither

Reply:
removed